apps/docs/src/contexts/DocsThemeContext.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  createContext,
+  Dispatch,
+  SetStateAction,
+  useContext,
+  useMemo,
+  useState,
+} from "react";
+
+interface DocsTheme {
+  theme: string;
+  mode: string;
+}
+
+interface DocsThemeContextValue {
+  docsTheme: DocsTheme;
+  setDocsTheme?: Dispatch<SetStateAction<DocsTheme>>;
+}
+
+const DEFAULT_THEME = {
+  theme: "pentahoPlus",
+  mode: "dawn",
+};
```

Comment:
Having a static `pentahoPlus-dawn` default is bad UX IMO.
Shouldn't we instead leverage the existing light/dark switcher, and keep this exclusively for the theme (`ds3`, `ds5`, `pentahoPlus`) instead?

![image](https://github.com/user-attachments/assets/56f76942-4c8d-4108-aef8-a086b8aaaf23)